db/schema.rb

```diff
@@ -338,6 +338,17 @@
     t.index ["original_file_id"], name: "index_pdf_files_on_original_file_id", unique: true
   end
 
+  create_table "proceeding_type_scope_limitations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
The changes to this file other than line 13 don't seem to belong to this PR.
Use `db:schema:load` to align your DB with the `schema.rb`